pkg/objstore/objstore.go

```diff
@@ -83,7 +83,14 @@ func UploadFile(ctx context.Context, logger log.Logger, bkt Bucket, src, dst str
 	}
 	defer runutil.CloseWithLogOnErr(logger, r, "close file %s", src)
 
-	if err := bkt.Upload(ctx, dst, r); err != nil {
+	fileSize := int64(-1)
```

Comment:
Maybe I'm missing something, but this initialization doesn't seem to be necessary as the `Stat` call must succeed.

Reply:
Yes, let's remove it